src/main/java/com/exactpro/th2/common/schema/grpc/configuration/GrpcRawFilterStrategy.java

```diff
@@ -16,15 +16,16 @@
 
 package com.exactpro.th2.common.schema.grpc.configuration;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.Data;
-
 import java.util.List;
 
-@Data
```

Comment:
The same as above about `@Data`

Reply:
FilterRoutingStrategy is used as a property type in FilterRoutingStrategy